e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -11,10 +11,12 @@
 // Group: @integrations
 
 import * as TIMEOUTS from '../../../fixtures/timeouts';
+import * as MESSAGES from '../../../fixtures/messages';
 
 describe('I18456 Built-in slash commands: common', () => {
     let user1;
     let user2;
+    let userGroup;
```

Comment:
Should directly assign empty array here.
```suggestion
    let userGroup = [];
```